docs/source/configuration/telemetry/exporters/tracing/datadog.mdx

```diff
@@ -251,3 +311,106 @@ telemetry:
+```yaml title="router.yaml"
+telemetry:
+  exporters:
+    tracing:
+      common:
+        # All requests will be traced and sent to the Datadog agent.
+        sampler: always_on
+```
+
+Datadog agent config:
+```yaml
+otlp_config:
+  traces:
+    probabilistic_sampling:
+      # Only 10 percent of spans will be forwarded to Datadog
+      sampling_percentage: 10
+```
+
+**I want the most performance from the Router and am not concerned with the APM view. I use metrics and traces to monitor my application.**
+
+Set the `sample` to a low value to reduce the number of traces sent to Datadog. Leave `preview_datadog_agent_sampling` to `false`.
+
+```yaml title="router.yaml"
+telemetry:
+  exporters:
+    tracing:
+      common:
+        # Only 10 percent of requests will be traced and sent to the Datadog agent. The APM view will only show a subset of total request data but the Router will perform better.
+        sampler: 0.1
+        preview_datadog_agent_sampling: false
+```
+
+### `sampler` (default: `always_on`)
+
+The `sampler` configuration allows you to control the sampling decisions that the router will make on its own and decrease the rate at which you sample, which can have a direct impact on your Datadog bill.
+
+```yaml title="router.yaml"
+telemetry:
+  exporters:
+    tracing:
+      common:
+        # Only 10 percent of spans will be forwarded to the Datadog agent. Experiment to find a value that is good for you!
+        sampler: 0.1
+```
+
+If you are using the Datadog APM viw then you should set `preview_datadog_agent_sampling` to `true` and adjust the `sampler` to the desired percentage of traces to be sent to Datadog.
+
+### `preview_datadog_agent_sampling` (default: `false`)
+
+The Datadog APM view relies on traces to generate metrics. For this to be accurate 100% of requests must be sampled and sent to the Datadog agent.
+To prevent ALL traces from then being sent to Datadog, you must set `preview_datadog_agent_sampling` to `true` and adjust the `sampler` to the desired percentage of traces to be sent to Datadog.
+
+```yaml title="router.yaml"
+telemetry:
+  exporters:
+    tracing:
+      common:
+        # Only 10 percent of spans will be forwarded from the Datadog agent to Datadog. Experiment to find a value that is good for you!
+        preview_datadog_agent_sampling: true
+        sampler: 0.1
+```
+
+<Note>
+
+    Using `preview_datadog_agent_sampling` will send ALL spans to the Datadog agent, but only the `sampler` percentage of them will be forwarded to Datadog. This means that your APM view will be correct at the cost of
+    the Router sending more spans to the Datadog agent. This will have an impact on the resource usage and performance of both the Router and Datadog agent.
+
+    If you are OK with your APM view only showing a subset of traces, then you can leave `preview_datadog_agent_sampling` to `false`, however it is recommended to rely on OTLP metrics to gain insight into the Router's performance.
+
+</Note>
```

Comment:
Move this content out of the Note, while also copying the Notes from the overview here.

```suggestion
Using `preview_datadog_agent_sampling` will send _all_ spans to the Datadog Agent, but only the percentage of traces configured by the `sampler` will be forwarded to Datadog. This means that your APM view will be accurate, but it will incur performance and resource usage costs for both the router and Datadog Agent to send and receive all spans.

If your use case allows your APM view to show only a subset of traces, then you can set `preview_datadog_agent_sampling` to `false`. You should alternatively rely on OTLP metrics to gain insight into the router's performance.

<Note>
    
- The router doesn't support [`in-agent` ingestion control](https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/?tab=java#in-the-agent).
    
- Configuring `traces_per_second` in the Datadog Agent will not dynamically adjust the router's sampling rate to meet the target rate.

</Note>
```